### PR TITLE
Updated video link in Readme.

### PR DESCRIPTION
**Pull request by marianunez**
#### Summary

Updated the video link in the Readme to point to youtube with the exact starting time instead of Google Drive that requires a video download.

Thanks @stevemudie for the link.

### CODE REVIEW

**Review by jfrerich — APPROVED** [review]
haha!  Awesome, @marianunez.  You are going to be famous :)

**Comment by jfrerich** [comment]
@mickmister, this is a non-coding change and super simple.  We shouldn't need QA, but I did add you for a second review!

**Review by mickmister — APPROVED** [review]
Nice!

**Pull request merged**
